includes/admin/class-ess-admin-assets.php

```diff
@@ -67,6 +68,7 @@ public function admin_scripts() {
 		$suffix    = defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ? '' : '.min';
 
 		// Register Scripts
+		wp_register_script( 'metabox-js', ESS()->plugin_url() . '/assets/js/admin/metabox' . $suffix . '.js', array( 'jquery' ), '3.5.4' );
```

Comment:
Should be like `wp_register_script( 'ess-admin-meta-boxes', ESS()->plugin_url() . '/assets/js/admin/meta-boxes' . $suffix . '.js', array( 'jquery' ), ESS_VERSION );`